immich/upload.py

```
@@ -0,0 +1,272 @@
+
+    if delete_duplicates:
+        for filepath, _ in duplicates:
+            to_delete.append(filepath)
+
+    for filepath in to_delete:
+        if dry_run:
+            logger.info(f"Would have deleted {filepath}")
+            continue
+        try:
+            filepath.unlink()
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)